### ISSUE TITLE
5.1.0 Fontawesome Pro Js Files Folder structure

### ISSUE THREAD

**Issue by designjudo** [FortAwesome/Font-Awesome#13471]
We're having a huge issue with how to serve these behind a firewall using Nodejs. Up until 5.1.0 it was working fine when you were distributing the individual js files with each release. With this one, there is only one file, so I can't explicitly call individual icons to each page. 

Why was the file/folder structure changed? Can you please release individual js files? This is ideal for build within a Node/React infrastructure.

Here's the example that was working before this release:

```
import { faPlus } from 'local-package'
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'

<FontAwesomeIcon icon ={ faPlus } />
```

This new version is a breaking change, and I have multiple projects using your icon library.

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community.

Yes, there are some breaking changes in FA 5.1.0

Please check our upgrading guide: https://github.com/FortAwesome/Font-Awesome/blob/master/UPGRADING.md#50x-to-510

**Comment by designjudo** [comment]
Thank you for quickly responding, however this link or your reply do not provide a solution to my issue. As previously stated, we are behind a firewall, so do not have access to your new packages. My workaround for this prior to this release was building a package with the js files you were providing. Now that you aren't providing them anymore, I can't update past 5.0.13. 

In order to solve this, I either need individual js files as you did before, or documentation how I can explicitly call individual icons with react-fontawesome with your new 1 file solution.

Expecting us to pull down 985 icons when I only need a handful isn't an acceptable approach in terms of build size. The previous solution you provided was perfect. I can't work with this unless you have better documentation.

**Comment by tagliala** [comment]
Sorry, I'm having troubles in understanding this issue

@mlwilkerson do you have some suggestions?

**Comment by designjudo** [comment]
@InsanityMeetsHH Yes, I did read that. And sorry, that's incorrect as I'm using the Pro version. Also, the original post states that my teams can't access the scoped packages from behind a firewall.

**Comment by sensibleworld** [comment]
Thanks for opening this issue! Since it’s been a while and things have changed quite a bit in Font Awesome since this was originally posted, we’re closing it out to help keep our issue tracker focused on current work.

If this is still important for your project, please feel free to re-open or file a fresh issue with updated details. Thanks so much!
